forest/src/cli/mod.rs

```diff
@@ -24,16 +24,29 @@ pub(super) fn cli() -> Result<Config, io::Error> {
                 .takes_value(true)
                 .help("A toml file containing relevant configurations."),
         )
+        .arg(
+            Arg::with_name("genesis")
+                .long("genesis")
+                .takes_value(true)
+                .help("The genesis CAR file"),
+        )
         .get_matches();
 
+    let mut cfg: Config = Config::default();
+
     if let Some(config_file) = app.value_of("config") {
         // Read from config file
         let toml = read_file_to_string(config_file)?;
 
         // Parse and return the configuration file
-        return Ok(read_toml(&toml)?);
+        cfg = read_toml(&toml)?;
     };
```

Comment:
Optional: Can just set cfg based on an if/else instead of setting and resetting in the if statement

Reply:
matched instead